tests/test_flake8_async.py

```diff
@@ -399,6 +406,8 @@ def _parse_eval_file(
 
             if err_code == "error":
                 err_code = test
+            if only_parse_args and err_code + alt_code not in ERROR_CODES:
+                continue
```

Comment:
oops, I should rename the variable. This tries to parse the error code now in order to enable the checks in `assert_expected_errors`